### PR TITLE
[debian] Install nnstreamer shared object file in libdir

### PR DESCRIPTION
**Pull request by gichan-jang**
Install nnstreamer shared object file in libdir.
This fixes build fail of the nnstreamer-api.
https://code.launchpad.net/~gichan-jang/+recipe/ml-api-daily

Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

File: debian/nnstreamer-core.install [hunk 1]
```diff
@@ -4,4 +4,5 @@
 /usr/lib/nnstreamer/decoders/libnnstreamer_decoder_image_labeling.so
 /usr/lib/nnstreamer/decoders/libnnstreamer_decoder_direct_video.so
 /usr/lib/nnstreamer/filters/libnnstreamer_filter_cpp.so
+/usr/lib/*/libnnstreamer.so
```
Comment (dongju-chae) [inline debian/nnstreamer-core.install:7]: Why don't you use `nnstreamer-core.links`

Comment (gichan-jang) [reply debian/nnstreamer-core.install]: Thanks. Changed!

Comment (dongju-chae) [reply debian/nnstreamer-core.install]: Uh.. I didn't see https://github.com/nnstreamer/nnstreamer/commit/1d1c9926bd3277a1724b3f52ea3bffb31422bc52

**Review by dongju-chae — APPROVED** [review]
LGTM

**Review by kparichay — APPROVED** [review]
LGTM

**Review by zhoonit — APPROVED** [review]
LGTM!

**Pull request merged**